cisco_secure_web_appliance/assets/logs/cisco-secure-web-appliance.yaml

```diff
@@ -0,0 +1,438 @@
+id: cisco-secure-web-appliance
+metric_id: cisco-secure-web-appliance
+# If for some reason id must be different than app_id, add the app_id in this field instead.
+# If id and app_id already match, this field can be left blank.
```

Comment:
Nit: We can probably remove this comment here

Reply:
Updated as Suggested.